components/managed-identity/interpolated-defaults.tf

```diff
@@ -10,7 +10,7 @@ data "azurerm_resource_group" "genesis_rg" {
 
 locals {
 
-  environment = (var.environment == "perftest") ? "test" : (var.environment == "aat") ? "stg" : (var.environment == "ptlsbox") ? "sbox" : "${var.environment}"
+  environment = (var.environment == "perftest") ? "test" : (var.environment == "aat") ? "stg" : (var.environment == "ptlsbox") ? "sbox" : (var.environment == "ptl") ? "prod" : "${var.environment}"
```

Comment:
Why is it being changed to prod?

Reply:
This is changed due to the KV that the managed identity is calling from, being in the azure-control-prod-rg RG.